lib/matplotlib/transforms.py

```diff
@@ -1548,12 +1552,18 @@ def __str__(self):
             return str(self._child)
 
     def __getstate__(self):
-        # only store the child
-        return {'child': self._child}
+        # only store the child and parents
+        return {
+            'child': self._child,
+            # turn the weakkey dictionary into a normal dictionary
+            'parents': dict(six.iteritems(self._parents))
+        }
 
     def __setstate__(self, state):
         # re-initialise the TransformWrapper with the state's child
-        self.__init__(state['child'])
+        self._init(state['child'])
+        # turn the normal dictionary back into a WeakValueDictionary
+        self._parents = WeakValueDictionary(state['parents'])
```

Comment:
This is the same method used in the grandparent `TransformNode`, so if it were going to fail, it would have done so a while ago.

There should be a strong reference from parent -> child assuming the transform's implemented correctly.